src/plugins/intel_cpu/src/nodes/input.cpp

```diff
@@ -350,10 +351,19 @@ void Input::cloneBlobIfRequired() {
                 + "_" + ptr;
     };
 
+
+    bool needFlushDenormalsToZero;
+    if (is_denormals_as_zero_set()) {
```

Comment:
BTW, we have some fundamental issue here. The check itself only shows us, that at the current moment of time the DAZ flag is set in the control register. But, this does not guaranty that the state will not be changed later. I mean we take the decision based only on the temporal state. Should not we check some more fundamental state here? For example the plugin control flag?

Reply:
Do you mean "CPU_DENORMALS_OPTIMIZATION"? this flag actually only will be effective at plugin "LoadExeNetwor" stage and there is no other place changes the flag,  thus by the time of `cloneBlobIfRequired()` , we can tell the flag by read DAZ MSR. also user app may set DAZ w/o relying on "CPU_DENORMALS_OPTIMIZATION" flag, it's actually quite simple: https://oneapi-src.github.io/oneDNN/dev_guide_data_types.html?highlight=denormal#floating-point-environment



Also considering the fact: DAZ flag will be inherited by all threads created after DAZ is set, and there is no easy way to change each thread's own DAZ in coherent way. I think a reasonable assumption is  once user is sure DAZ does no harm to accuracy,  the user app will set DAZ only once at the beginning of the application and never touch that flag anymore during the whole life-span of the application.

based on this assumption we can check DAZ flag here.

If you concern customer application may change the DAZ dynamically (which is crazy to me), then we have no way to know what the flag will be set at inference time in advance at load network stage (where `cloneBlobIfRequired()` is called).

Reply:
The subject here is a separation of responsibility. When an user app set or unset the DAZ and FTZ flags it is a problem of the user app. But I mean our own code. Technically the DAZ flag can be manipulated from any place of OV code. The only sing the CPU plugin is responsible for in this context is the 'CPU_DENORMALS_OPTIMIZATION' parameter, which adds some responsibility to the plugin. That means that if in general case we see DAZ flag set and CPU_DENORMALS_OPTIMIZATION is OFF, that probably the flag was set in some other place of the application and probably is not controlled by the plugin. My question is whether we can use the flag as an indication of the specific plugin state, which clearly defines the fact that we can skip deformalized values nullification. Does it make sense?

Reply:
Actually I even think we shouldn't clear denormals at all unless user explicitly required so(by set some other flags for example). Current solution is not transparent to customer in my opinion.

since we now have CPU_DENORMALS_OPTIMIZATION  flags, I suspect that we may even drop this internal ftz behavior at all.

What do you think?

Reply:
Yes implicit denormals nullifying is not obvious to the user. And when we properly validate the DAZ and FTZ flags usage (my main concern here is preserving the previous program behavior) we should remove this denormals related preprocessing.

Reply:
So yes. Probably we can consider this PR as an intermediate state.
But could you please answer, can we use CPU_DENORMALS_OPTIMIZATION flag check instead of the looking at the CPU control register?

Reply:
Yes, we can ,will try that approch